### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by t-book**
Fix several typos across all files.

### CODE REVIEW

**Comment by t-book** [comment]
@gannebamm I know this is quite big PR done with https://github.com/tbroadley/github-spellcheck-cli may I ask you for a quick check?

**Comment by gannebamm** [comment]
Today I am busy but Sunday or at least Monday I will take a look

**Comment by t-book** [comment]
@gannebamm Danke! But this is not urgent at all! So there is no need to work on weekends ;)

File: admin/default_lang/index.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Unfortunately  Django overrides this setting, giving the language setting of a u
 Additional Steps
 ================
 
-If this is not the desired behaviour, and all users should initially see the default ``LANGUAGE_CODE``, regardless of their browser's settings, do the following steps to ensure Django ignores the browser language settings.  (Users can always use the pulldown language menu to change the language at any time.)
+If this is not the desired behavior, and all users should initially see the default ``LANGUAGE_CODE``, regardless of their browser's settings, do the following steps to ensure Django ignores the browser language settings.  (Users can always use the pulldown language menu to change the language at any time.)
```
Comment (gannebamm) [inline admin/default_lang/index.rst]: https://ell.stackexchange.com/questions/202548/what-is-the-difference-between-behavior-and-behaviour

File: admin/upgrade/index.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ Upgrade from 2.4.x
 ==================
 
 These are the notes of a migration from 2.4.x to 2.10.1.
-These notes could possibly works also when migrating from 2.6.x, 2.7.x, 2.8.x but were not tested in that scenarios.
+These notes could possibly works also when migrating from 2.6.x, 2.7.x, 2.8.x but are not tested in that scenarios.
```
Comment (gannebamm) [inline admin/upgrade/index.rst]: ~~works~~ work

File: advanced/contrib/index.rst [hunk 3]
```diff
@@ -168,7 +168,7 @@ whose memberships will not be re-evaluated.
 
 .. note:: Users mapped from LDAP will be marked with an ``ldap`` tag. This will be used to keep them in sync.
 
-.. warning:: If you remove the ``ldap`` tag, the users will be threated as pure internal GeoNode ones.
+.. warning:: If you remove the ``ldap`` tag, the users will be threaten as pure internal GeoNode ones.
```
Comment (gannebamm) [inline advanced/contrib/index.rst]: ~~threaten~~ 😆 
>> treated

File: install/core/index.rst [hunk 4]
```diff
@@ -99,7 +99,7 @@ It will run locally against a file-system based ``SQLite`` database.
 
 First of all we need to prepare a new Python Virtual Environment
 
-Since geonode needs a large number of different python libraries and packages, it's recommended to use a python virtual environment to avoid conflicts on dependencies with system wide python packages and other installed software. See also documentation of `Virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/stable/>`_ package for more information
```
Comment (gannebamm) [inline install/core/index.rst]: it is

**Pull request merged**